src/nfts/NftMedia.tsx

```diff
@@ -157,6 +157,7 @@ export default function NftMedia({
             minLoadRetryCount={3}
             repeat={true}
             resizeMode={shouldAutoScaleHeight ? 'contain' : 'cover'}
+            muted={true}
```

Comment:
On iOS this works, unfortunately on Android this appears to remove all volume control and the media is always muted.

Reply:
`muted={Platform.OS === 'ios'}` will fix this for iOS, but I am not sure what to do about Android.

Reply:
i just noticed it doesn't quite work as expected on iOS too, i made some notes in the task. i think the problem with Android is the default controls this lib provides doesn't include anything for volume :/ seems like we need to do something extra to get volume control. i'll close this PR for now, seems non urgent and better to address both platforms at once